paper-checkbox.html

```diff
@@ -274,7 +274,17 @@
           var inkSize = this.getComputedStyleValue('--calculated-paper-checkbox-ink-size').trim();
           // If unset, compute and set the default `--paper-checkbox-ink-size`.
           if (inkSize === '-1px') {
-            var checkboxSize = parseFloat(this.getComputedStyleValue('--calculated-paper-checkbox-size').trim());
+            var checkboxSizeText = this.getComputedStyleValue('--calculated-paper-checkbox-size').trim();
+
+            // Is the value not in units of 'px'?
```

Comment:
I would explain here _why_ we need to do this, because it's really not obvious why we can't do the `calc` in all cases